torchvision/prototype/transforms/_utils.py

```diff
@@ -20,7 +20,7 @@ def fn(
     try:
         return next(query_recursively(fn, sample))[1]
     except StopIteration:
-        raise TypeError("No image was found in the sample")
+        raise TypeError("No image was found in the sample") from None
```

Comment:
Why do you "from None" here ?

Reply:
Sorry, forgot to add a comment. The `StopIteration` is an internal detail that does not need to be propagated outside. By adding `from None` the error will look like a normal exception

```py
try:
    next(iter([]))
except StopIteration:
    raise TypeError("Argh!") from None
```

```
TypeError: Argh!
```

Not doing that gives

```py
try:
    next(iter([]))
except StopIteration:
    raise TypeError("Argh!")
```

```
StopIteration

During handling of the above exception, another exception occurred:

[...]
TypeError: Argh!
```